src/data-access/LexemeCreator.ts

```diff
@@ -14,6 +14,6 @@ export default interface LexemeCreator {
 		lemmaLanguageCode: string,
 		lexemeLanguageItemId: string,
 		lexicalCategoryItemId: string,
-	): Promise<string>;
+	): Promise<URL>;
```

Comment:
The `@return` jsdoc should be updated, now that we return a URL and not a lexeme ID.